server/models/service-process.js

```diff
@@ -333,6 +333,34 @@ module.exports = function extendServiceProcess(ServiceProcess) {
   ServiceProcess.prototype.heapSnapshot = heapSnapshot;
 
   /**
+   * Check to see if a specific capability is supported.
+   *
+   * @param {string} feature Feature to check.
+   * @param {function} callback Callback function.
+   */
+  function queryCapabilities(feature, callback) {
+    this._appCommand({
+        cmd: 'query-capabilities',
+        target: this.pid,
+        feature: feature,
+      }, callback);
+  }
+  ServiceProcess.prototype.queryCapabilities = queryCapabilities;
+
+  /**
+   * Check to see if all capabilities that are supported.
```

Comment:
Caller's of the API would probably thank you for documenting the format of what's returned (not that our API is a paragon of thorough documentation). Maybe just inject some example output.